man/systemd-system.conf.xml

```diff
@@ -171,6 +171,126 @@
         effect if a hardware watchdog is not available.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>IOCostQOSPath=</varname></term>
+        <listitem><para> Configure the absolute path to device to be configured
+        in io.cost.qos file of the root cgroup.</para></listitem>
```

Comment:
hmm, root cgroup?

this is user facing configuration, please don#t explain how this is implemented, but explain what this is used for and how to use it. i.e. names of lower-level cgroup attributes should maybe be mentioned as additional info, but certainly not as primary explanation, because that's just confusing and not helpful. Unless someone actually knows the iocost cgroup controller first, it'll not help at all.